### DISCUSSION TITLE
アバウトセクションの文章をつめていく

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#181] [00: 未分類]
## 現状
STEP1~8までの簡単な記述

## 理想の状態
- よりわかりやすく、明るく
  - 絵文字や画像を入れる

## 関連リンク
#173

**Comment by kazzyfrog** [comment]
詳細は、READMEに記載してあるので、めっちゃ簡潔に出来たら良さそうですよね！
１セクションにつき、
見出し＋２〜３分の説明＋画像とか絵文字とか、、！

**Reply by kagomen** [reply]
ですね！

最低限のセクション候補として、

- そもそもこのプロジェクトは何なのか
  - （プロジェクトなのかチュートリアルなのか・・・？）
- 参加者は何をするのか
  - （データを書き換える）
- 何（↑）をしたらどうなるのか
  - （LGTM画像が生成されて、トップページに絵文字などが反映される、など）
  - （見出しタイトルが難しい・・・）

の3点は必要かなと思うのですが、これはいらないよ、とか、他に何か書いた方が良いこととかってありますかね…？

**Comment by kazzyfrog** [comment]
現在の仮デザイン案に合わせて、
アバウトセクションの文章とざっくりデザインの感じなど、自分の方でも詰めてみます！

**Reply by kazzyfrog** [reply]
セクション候補として、以下の感じではどうでしょうか？
ざっくりという感じですが、、

- （ヒーロー）
- コンセプト（問題意識）
  - Git/GitHubを勉強しても、1人だと使う必要性があまりなくて、結局忘れてしまう。。
  - そもそもGitHubは、共同開発や、それに伴うバージョン管理を行うツールなので、1人で勉強しているとイマイチ掴めない。。
  - そんな問題を解決する、誰でも参加できるように設計されたオープンなプロジェクトです。チュートリアルを行うことで、GitHubを使った共同開発を実際に体験して学べます！
- チュートリアル（概要）
  - 私たちは、あなたが見ているこのWebサイトを、今よりもっとPOPにしたいです！
  - あなたの力を貸してください！
  - 具体的な作業の内容は、簡単です。３０分もあれば完了します。
  - ぜひREADMEをチェックしてみてください。
  - チュートリアルを完了したら、このサイト上に、あなたが入力した絵文字と一言メッセージが表示されます。
- 初めてのコントリビューションを体験する
  - これは同時に、オープンソースのプロジェクトに参加する方法についても学ぶことができます。
  - チュートリアルを完了したら、参加者の個別ページが作成されます。
  - GitHub上のテキストコミュニケーションを、少しでも楽しいものにするための、オリジナルのLTGM画像が生成されるので、ぜひ使ってみてください！
  - HTML, CSS or JSXはかけますか？このLGTM画像のテーマもあなたが開発できますよ！

**Reply by kagomen** [reply]
すみません、返事が遅くなりました……！🙇‍♀️

「このプロジェクトが何なのか」って、なんだか説明しにくいなーと思ってたのですが、
問題意識の提示、めちゃくちゃ良いと思います！👀✨　すごいです！

> 私たちは、あなたが見ているこのWebサイトを、今よりもっとPOPにしたいです！

これもGOODすぎます……！
これだと8ステップが省けるし、冗長にならずに済みますね……！

> HTML, CSS or JSXはかけますか？このLGTM画像のテーマもあなたが開発できますよ！

この誘導も、すごく良いと思います！！
考えてくださり、ありがとうございますー！🙏✨
